hackage-security/src/Hackage/Security/TUF/Snapshot.hs

```diff
@@ -76,9 +77,12 @@ instance ( MonadReader RepoLayout m
     snapshotVersion     <- fromJSField enc "version"
     snapshotExpires     <- fromJSField enc "expires"
     snapshotMeta        <- fromJSField enc "meta"
-    snapshotInfoRoot    <- FileMap.lookupM snapshotMeta (pathRoot       repoLayout)
-    snapshotInfoMirrors <- FileMap.lookupM snapshotMeta (pathMirrors    repoLayout)
-    snapshotInfoTarGz   <- FileMap.lookupM snapshotMeta (pathIndexTarGz repoLayout)
+    let lookupMeta k = case FileMap.lookup k snapshotMeta of
```

Comment:
Why not put `lookupMeta` in `TUF.FileMap`?

Reply:
I mostly wanted to avoid increasing the exported API surface with a trivial definition; after all, having something like  `lookupM` exposed was what had been causing this headache in the first place...